Build-your-first-dApp.md

```diff
@@ -42,7 +42,7 @@ If you would rather learn from a video, we have a recording available of this tu
       - [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) - Code formatter for JS, CSS, and HTML files
       - [npm IntelliSense](https://marketplace.visualstudio.com/items?itemName=christian-kohler.npm-intellisense) - Autocompletes npm modules in import statements
       - [IntelliSense for CSS class names in HTML](https://marketplace.visualstudio.com/items?itemName=Zignd.html-css-class-completion) - Autocomplete for CSS classes in HTML files
-      - [gitLeens](https://marketplace.visualstudio.com/items?itemName=naco-siren.githlens) - Shows git blame information in the editor
+      - [GitLens](https://marketplace.visualstudio.com/items?itemName=naco-siren.githlens) - Shows git blame information in the editor
```

Comment:
Gotta change the link as well mate 😄 

https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens